### ISSUE TITLE
Operationalize app

### ISSUE THREAD

**Issue by shaidar** [mitodl/library#1]
Steps required to deploy app in a production environment:
- [x] Request new Google Drive from IS&T
- [x] Setup Heroku pipeline
- [x] Add RC app
- [x] Fork [`nytimes/library-customization`](https://github.com/nytimes/library-customization-example) repository to use for overriding some behavior
- [ ] Look into triggering automatic build when commits happen on forked `library-customization` repo

**Comment by shaidar** [comment]
- Request new Google Drive from IS&T:
Peter requested a new drive from IS&T.

- Add RC app
RC app was added but since domain name is different from production, need to setup a new OAuth client.

- Triggering automatic build
Customizations will be taking place in the forked `library-customization` repo but since that's not the main repo that the Heroku app is using, need to find a way to trigger a build when a change happens in that other repo.

**Comment by shaidar** [comment]
I came across a discussion (maybe google groups) which I can't find anymore related to triggering an automatic deployment when changes are merged into the `library-customization` repo. In reality a dyno restart suffices as it runs the shell script - https://github.com/mitodl/library/blob/main/bin/install_customizations on restart. Not sure it's worth adding some kinda webhook functionality given that once the app is running, there will probably be little customizations taking place. We did make a minor tweak (https://github.com/mitodl/library/commit/e962fc877a02c85059eeb141314feda323c37c96) to the script so that we can set a specific branch of the repo. That way we can have the production app running off of the `release` branch and QA/RC using the main one in case we're trying out some things. 

I'm closing this issue then and if we ever decide that we need some kinda webhook, then we can open another/separate issue for that work.
